src/test/java/gov/cms/mat/cql_elm_translation/service/CqlConversionServiceTest.java

```diff
@@ -182,11 +185,20 @@ void testProcessCqlDataWithErrorsMissingModel() {
   @Test
   void testGetTranslatedLibrariesForCqlForCql() throws IOException {
     String cql = getData("/qdm_data_criteria_retrieval_test.cql");
+    String matGlobal = getData("/mat_global_common_functions.cql");
+    MadieLibrarySourceProvider.setUsing(new CqlTextParser(cql).getUsing());
+    MadieLibrarySourceProvider.setCqlLibraryService(cqlLibraryService);
+    doReturn(matGlobal)
+        .when(cqlLibraryService)
+        .getLibraryCql(eq("MATGlobalCommonFunctions"), eq("7.0.000"), nullable(String.class));
+
     List<TranslatedLibrary> libraries = service.getTranslatedLibrariesForCql(cql, "token");
     AtomicBoolean foundAMatch = new AtomicBoolean();
-    libraries.forEach(
-        library -> foundAMatch.set(library.getElmJson().contains("DataCriteriaRetrivalTest")));
-    assertThat(foundAMatch.get(), is(true));
+    var matchingLib =
+        libraries.stream()
+            .filter(library -> library.getElmJson().contains("DataCriteriaRetrivalTest"))
+            .findFirst();
+    assertThat(matchingLib.get().getName(), is(equalTo("DataCriteriaRetrivalTest")));
```

Comment:
unrelated to this ticket. I will revert if anyone objects. this is to fix the gundam build failure: https://qnetjenkins.cms.gov/bmat-master/job/Madie/job/Madie%20Services/job/cql-elm-translator-service/job/sprint%252Fgundam_2024-01-29/3/console.